frontend/src/components/SidebarPlugin/SideBar.vue

```diff
@@ -114,6 +111,11 @@ export default {
     logout() {
       this.$emit('logout')
     },
+    getElopageLink() {
+      return this.$store.state.hasElopage
+        ? `https://elopage.com/s/gradido/sign_in?locale=${this.$i18n.locale}`
```

Comment:
@Mogge: Should this also be encoded?
```suggestion
        ? encode(`https://elopage.com/s/gradido/sign_in?locale=${this.$i18n.locale}`)
```